deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -132,6 +131,36 @@ spec:
               name: cni-bin-dir
             - mountPath: /host/etc/cni/net.d
               name: cni-net-dir
+{{- if .Values.repair.enabled }}
+        # This container repairs broken CNI pods
+        - name: repair-cni
+          image: {{ .Values.repair.hub }}/install-cni:{{ .Values.repair.tag }}
+          imagePullPolicy: {{ .Values.pullPolicy }}
+          command:
+          - /opt/cni/bin/istio-cni-repair
```

Comment:
The changes to implement and build that command have not been backported to `istio/cni:release-1.4`. That must be done first, or in this same PR.

Reply:
Yep, I meant for this to be a WIP PR. Added the tag.